### DISCUSSION TITLE
Feed data from server

### DISCUSSION THREAD

**Discussion by GianfraGovUK** [FlowmapBlue/FlowmapBlue#71] [Q&A] [answered]
Hello, thank you for this amazing project!
I wanted to explore the possibility to feed it from our server as we have data we cannot share outside our organisation. Do you know of any development in this direction?
Many thanks

Gianfranco

**Comment by ilyabo** [comment, marked as answer]
Hi Gianfranco,

I am actually working on a commercial version of the tool which supports secure data storage. Please contact me at ilyabo AT gmail.com if you are interested.
